test/Kestrel.Core.Tests/ReasonPhrasesTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Tests
+{
+    public class ReasonPhraseTests
+    {
+        [Theory]
+        [InlineData(999, "Unknown")]
+        [InlineData(999, null)]
```

Comment:
Could put the entire expected output in the InlineData? Then you could just do `Assert.Equal(expectedString, Encoding.ASCII.GetString(bytes))`.

Reply:
done.